components/apimgt/org.wso2.carbon.apimgt.micro.gateway/org.wso2.carbon.apimgt.micro.gateway.configurator/src/test/resources/carbon-home/repository/conf/registry.xml

```diff
@@ -0,0 +1,370 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
+  ~ Copyright 2005-2011 WSO2, Inc. (http://wso2.com)
+  ~
```

Comment:
Update the year.

Reply:
These are not files that I have implemented, this is maintained as a resource for testing purposes. So, updating the license year is not required IMO.